RecoCTPPS/PixelLocal/plugins/CTPPSPixelRecHitProducer.cc

```diff
@@ -0,0 +1,49 @@
+#include "RecoCTPPS/PixelLocal/interface/CTPPSPixelRecHitProducer.h"
+
+
+CTPPSPixelRecHitProducer::CTPPSPixelRecHitProducer(const edm::ParameterSet& conf) :
+  param_(conf), cluster2hit_(conf)
+{
+  src_ = conf.getParameter<std::string>("label");
+  verbosity_ = conf.getUntrackedParameter<int> ("RPixVerbosity");	 
+  tokenCTPPSPixelCluster_ = consumes<edm::DetSetVector<CTPPSPixelCluster> >(edm::InputTag(src_));
+  produces<edm::DetSetVector<CTPPSPixelRecHit> > ();
+}
+
+CTPPSPixelRecHitProducer::~CTPPSPixelRecHitProducer() {}
+
+void CTPPSPixelRecHitProducer::fillDescriptions(edm::ConfigurationDescriptions & descriptions){
+  edm::ParameterSetDescription desc;
+  desc.addUntracked<int>("RPixVerbosity",0);
+  desc.add<std::string>("label", "ctppsPixelClusters");
```

Comment:
Better to allow already an InputTag in the configuration, rather than a string to be converted in the constructor

Reply:
I think I have modified it accordingly to your suggestion. Please check.